### PR TITLE
PE-485 Devstack setting from common file

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description 
Devstack setting from common file

### CODE REVIEW

File: openedx/core/djangoapps/ace_common/apps.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class AceCommonConfig(AppConfig):
             ProjectType.CMS: {
                 SettingsType.AWS: {PluginSettings.RELATIVE_PATH: u'settings.aws'},
                 SettingsType.COMMON: {PluginSettings.RELATIVE_PATH: u'settings.common'},
-                SettingsType.DEVSTACK: {PluginSettings.RELATIVE_PATH: u'settings.devstack'},
+                SettingsType.DEVSTACK: {PluginSettings.RELATIVE_PATH: u'settings.common'},
```
Comment (Squirrel18) [inline openedx/core/djangoapps/ace_common/apps.py:22]: Can we simply remove the configuration type? @andrey-canon

Comment (andrey-canon) [reply openedx/core/djangoapps/ace_common/apps.py:22]: from line 19 to 22 ??? yes, this is only code from the backport but I think it is not necessary @Squirrel18

Comment (Squirrel18) [reply openedx/core/djangoapps/ace_common/apps.py:22]: No, I mean, can we remove the DEVSTACK setting type, we are not using it. @andrey-canon

Comment (andrey-canon) [reply openedx/core/djangoapps/ace_common/apps.py:22]: the line defines the plugin setting for devstack, but the settings are been defined  with a different file, it doesn't mean that the settings are not been using @Squirrel18

Comment (Squirrel18) [reply openedx/core/djangoapps/ace_common/apps.py:22]: If the DEVSTACK settings are not defined, the platform will use a COMMON settings, right? @andrey-canon

Comment (andrey-canon) [reply openedx/core/djangoapps/ace_common/apps.py:22]: I don't think so, but the true is that I don't know @Squirrel18

Comment (Squirrel18) [reply openedx/core/djangoapps/ace_common/apps.py:22]: Jajaja ok.

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**